makefile

```diff
@@ -3,21 +3,21 @@ VERSION := $(shell cat build/manifest.json | jq .version)
 
 .PHONY: firefox
 firefox:
-	# ifneq (,$(wildcard blue-blocker-firefox-$(VERSION).zip))
-	# 	rm "blue-blocker-firefox-${VERSION}.zip"
-	# endif
+# ifneq (,$(wildcard blue-blocker-firefox-$(VERSION).zip))
```

Comment:
Changes in this file are just so these lines aren't printed when ran. Indented comments are printed while un-indented ones are not.